app/resonant-laboratory/views/widgets/DatasetView/template.html

```diff
@@ -1,5 +1,5 @@
-<img id="noDatasetState" class="emptyStateHelper" src="../../../images/emptyState/noDataset.svg"/>
-<div id="datasetOverview">
+<img id="noDatasetState"  style="display: none;" class="emptyStateHelper" src="../../../images/emptyState/noDataset.svg"/>
```

Comment:
I agree, but what's the alternative? The problem is that a bunch of jumbled stuff shows up as soon as the template is instantiated, then disappears when the appropriate show()/hide() calls go occur.